packages/playback-core/src/index.ts

```diff
@@ -691,13 +691,57 @@ export const loadMedia = (
     }
   };
 
+  let prevSeekableStart: number;
+  let prevSeekableEnd: number;
+
+  const seekableChange = () => {
+    const nextSeekableStart = getSeekable(mediaEl)?.start(0);
+    const nextSeekableEnd = getSeekable(mediaEl)?.end(0);
+    if (prevSeekableEnd !== nextSeekableEnd || prevSeekableStart !== nextSeekableStart) {
+      mediaEl.dispatchEvent(new CustomEvent('seekablechange', { composed: true, bubbles: true }));
```

Comment:
should we keep this the same to native video events?
bubbles is false, composed false also

Reply:
We'll need `composed` to be `true` for it to be available to e.g. Media Chrome/`MediaStore` (or do a bunch of extra work to delegate.

`bubbles` can definitely be `false`, and would be consistent with the recent (**_finally_**) updates to our Media Chrome state change events. I used `true` because that's what we use for all of our other custom events in elements, but happy to start undoing that pattern/habit, starting with `seekablechange`.

Reply:
ah k I see, then probably best to leave it for now. bubbles might be needed too for it to go from <video> to <mux-video> or does composed handle that...

Reply:
that's... a good question. I'll test that now.

Reply:
for archival purposes - `bubbles` not needed for shadow DOM boundary propagation (only `composed`), so removing. 🎉